src/fd_monitor.cpp

```diff
@@ -199,7 +199,7 @@ void fd_monitor_t::run_in_background() {
     }
 }
 
-void fd_monitor_t::poke_in_background(const poke_list_t &pokelist) {
+void fd_monitor_t::poke_in_background(poke_list_t &&pokelist) {
```

Comment:
I think these two are also better as pass-by-value, per https://google.github.io/styleguide/cppguide.html#Rvalue_references. I am not a language lawyer but there should be no behavior difference since we always call `std::move` when calling this function, right?